packages/jest-core/tsconfig.json

```diff
@@ -4,14 +4,14 @@
     "rootDir": "src",
     "outDir": "build"
   },
-  // TODO: This is missing `@jest/reporters`
```

Comment:
should have done this when I merged #7994 yesterday 😅 The typings disagreed as well, interestingly enough (which is why it raised an error now)